packages/astro/src/cli/index.ts

```diff
@@ -60,6 +60,7 @@ async function printVersion() {
 function resolveCommand(flags: Arguments): CLICommand {
 	const cmd = flags._[2] as string;
 	if (cmd === 'add') return 'add';
+	if (cmd === 'docs') return 'docs';
 
 	if (flags.version) return 'version';
 	else if (flags.help) return 'help';
```

Comment:
I'd remove that `if` and add `'docs'` to the `supportedCommands` set. That what, if anyone does `astro docs --help`, they'll see the full help message!